template/types/action/fieldfilter.go

```diff
@@ -36,7 +37,9 @@ $("select` + jump.BtnId + `").on("select2:select",function(e){
 		let has = false;
 		for (let i = 0; i < vars.length; i++) {
 			pair = vars[i].split("=");
-			if (pair[0] === field) {
+			if (pair[0] == "` + parameter.Page + `") {
```

Comment:
Thanks for the pr👍👍. Maybe we can add some comments here which can help the developer read the code easily.